apps/cyberstorm-storybook/stories/components/Switch.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { Switch } from "@thunderstore/cyberstorm";
+import React, { useState } from "react";
+
+const meta = {
+  title: "Cyberstorm/Components/Switch",
+  component: Switch,
+} as ComponentMeta<typeof Switch>;
+
+const defaultArgs = {
+  state: false,
+};
+
+const Template: ComponentStory<typeof Switch> = (args) => {
+  return (
+    <div
+      style={{
```

Comment:
These styles on the other hand are copy pasted below, so maybe they could be defined only once, outside any templates. Also, I'm not sure if CSS variables can be used in these objects, but I would think they could? If so, maybe use e.g. the color values defined in globals.css (or whatever it's called now).